Wikipedia/Code/WMFAppViewController.m

```diff
@@ -240,7 +243,7 @@ - (void)configureArticleListController:(WMFArticleListTableViewController *)cont
 - (void)appWillEnterForegroundWithNotification:(NSNotification *)note {
     self.unprocessedUserActivity = nil;
     self.unprocessedShortcutItem = nil;
-    [self resumeApp];
+    [self resumeApp:^{}];
```

Comment:
I've been making completions "nonnull" to avoid the nil check, but ultimately end up doing the nil check, so probably could just make them nullable. In this case nil would be fine since the completion is nullable. In Swift I've also been making them non-optional.